edb/edgeql/compiler/context.py

```diff
@@ -484,6 +484,9 @@ class ContextLevel(compiler.ContextLevel):
     defining_view: Optional[s_types.Type]
     """Whether a view is currently being defined (as opposed to be compiled)"""
 
+    compiling_update_shape: bool
```

Comment:
It looks like this is dead? Nothing seems to use it

Reply:
Oops. Yes, it's part of an aborted attempt to condition the resolution of descendants in ptrrefs.